src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,25 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    public static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    public static final DateTimeFormatter DATE_TIME_FORMATTER =
+            DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm", Locale.ENGLISH);
+    public static final int TIME_OFFSET = 2;
+    LocalDate currentDate = LocalDate.now();
```

Comment:
let's make it a local variable in methods where we need it, this way it will be initialized every time method is called, not once when object of this class is created